src/script/engine.zig

```diff
@@ -353,6 +363,12 @@ pub const Engine = struct {
         try self.stack.pushByteArray(value);
     }
 
+    /// OP_ROT: The top three items on the stack are rotated to the left
+    fn opRot(self: *Engine) !void {
+        try self.stack.swap(0, 1);
+        try self.stack.swap(1, 2);
```

Comment:
same. You are rotating the bottom elements of the stack.
Add more elems to you test and it will fail